tensorflow/java/src/gen/resources/tftypes.csv

```diff
@@ -0,0 +1,21 @@
+TF type,datatype code,Java type,Java boxed type,Creator?,default,Description

```

Comment:
Personally, I feel that including it in the script as a datastructure would be easier to follow as it would allow both the definition and its use in the script to use named fields, but I don't feel too strongly right now (and perhaps it's prettier in Python than in Perl), so okay for now. I guess